plugins/sourcegraph/access_token.go

```diff
@@ -0,0 +1,64 @@
+package sourcegraph
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func AccessToken() schema.CredentialType {
+	return schema.CredentialType{
+		Name:    credname.AccessToken,
+		DocsURL: sdk.URL("https://docs.sourcegraph.com/cli"),
+		Fields: []schema.CredentialField{
+			{
+				Name:                fieldname.Username,
+				MarkdownDescription: "Username on your Sourcegraph instance.",
+				Secret:              false,
+				Composition: &schema.ValueComposition{
+					Charset: schema.Charset{
+						Lowercase: true,
+						Digits:    true,
+						Specific:  []rune{'.', '-', '_'},
+					},
+				},
+				Optional: true,
+			},
```

Comment:
Is there a reason for including the username here (even as optional), given the credential provisioner and importer do not provision/import this field at all?